<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>672</width>
    <height>479</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="OriImage_Disp">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>384</width>
      <height>288</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="TempImage_Disp">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>30</y>
      <width>200</width>
      <height>50</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>华文楷体</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>处理后效果图</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>0</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>华文楷体</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="text">
     <string>模板图像</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>370</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>选择文件夹</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>370</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>退出</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>370</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>开始</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>100</y>
      <width>261</width>
      <height>211</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>华文楷体</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="title">
     <string>输出结果</string>
    </property>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>80</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>华文楷体</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="text">
      <string>线数：</string>
     </property>
    </widget>
    <widget class="QLabel" name="Line_num">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>30</y>
       <width>131</width>
       <height>30</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::WinPanel</enum>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>111</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>华文楷体</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>接触点坐标：</string>
     </property>
    </widget>
    <widget class="QLabel" name="x_aix">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>70</y>
       <width>50</width>
       <height>30</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::WinPanel</enum>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="y_aix">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>70</y>
       <width>50</width>
       <height>30</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::WinPanel</enum>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>101</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>华文楷体</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>有无弓架：</string>
     </property>
    </widget>
    <widget class="QLabel" name="gong_is">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>110</y>
       <width>131</width>
       <height>30</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::WinPanel</enum>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>91</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>华文楷体</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>处理时间：</string>
     </property>
    </widget>
    <widget class="QLabel" name="process_time">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>150</y>
       <width>131</width>
       <height>30</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::WinPanel</enum>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>672</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>操作</string>
    </property>
    <addaction name="Hand_Templete"/>
    <addaction name="actionAuto_Templete"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="Hand_Templete">
   <property name="text">
    <string>手动分割模板</string>
   </property>
  </action>
  <action name="actionAuto_Templete">
   <property name="text">
    <string>自动分割模板</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
